backend/src/main/java/gov/cdc/usds/simplereport/db/model/TestOrder.java

```diff
@@ -55,6 +55,11 @@ public class TestOrder extends BaseTestInfo {
   @Setter
   private Set<Result> results = new HashSet<>();
 
+  @Column(name = "timer_started_at")
+  @Getter
+  @Setter
+  private String timerStartedAt;
```

Comment:
do we not want this to be a date?

Reply:
I could update this if you prefer but I don't think it necessarily has to be a Date object. This value is only being used by the frontend timer where it needs the value as epoch milliseconds anyway, as opposed to parsing the String into a Long and then into a Date just to convert it back again when that value is retrieved

Reply:
that makes sense. non-blocking followup question: do the other date values we use on the frontend need to be converted to epoch milliseconds (since they are stored as dates) or is this specific to the implementation of the timer?

Reply:
I think this is just specific to the implementation of the timer